provision/createcontrail

```diff
@@ -2,25 +2,23 @@
 case "$1" in
 	"")
 echo "Script usage:"
-echo "-c|--create Login Password ProjectID DoaminName ProjectName NetworkName PublicKey PrivateKey RouterIP Orchestrator Branch  - creating new instace "
+echo "-c|--create Login Password MachineName ProjectID DoaminName ProjectName NetworkName PublicKey PrivateKey RouterIP Orchestrator Branch flavor terraformStateFileName mainDirectoryName contrailType patchsetRef - creating new instace "
 echo "Orchestrator could be set to openstack or kubernetes "
 echo "Branch parameter is optional"
-echo "-d|--destroy Login Password ProjectID domainName ProjectName Orchestrator	- destroying instance"
+echo "-d|--destroy Login Password MachineName ProjectID domainName ProjectName Orchestrator	terraformStateFileName mainDirectoryName - destroying instance"
 ;;
     --create|-c)
 
 
-if [ -z "${12}" ]
+if [ -z "${13}" ]
```

Comment:
What happens below this line, with super long commands with massive duplication, has to go. It's very scary to touch any of this and it just begs for bugs. This should be refactored long ago. Are you up for the challenge? :)

Reply:
I moved this file to Jenkinsfile since copying all those arguments seems to be very distracting.